src/main/helper/ApiFormatter.ts

```diff
@@ -403,6 +406,18 @@ export const mapRespondents = (respondents: RespondentApiModel[]): Respondent[]
   });
 };
 
+export const mapRepresentatives = (representatives: RepresentativeApiModel[]): Representative[] => {
+  if (representatives === undefined) {
+    return;
+  }
+  return representatives.map(rep => {
```

Comment:
This can be shortened to just `return respresentatives?.map` . Using the `?` here will return nothing if representatives if undefined